openmp/libomptarget/src/private.h

```diff
@@ -253,6 +253,19 @@ struct TargetMemcpyArgsTy {
         DstOffsets(DstOffsets), SrcOffsets(SrcOffsets),
         DstDimensions(DstDimensions), SrcDimensions(SrcDimensions){};
 };
+
+struct TargetMemsetArgsTy {
+  /**
```

Comment:
style

Reply:
Usually the comment for a class is outside of the class

Reply:
Resolved this one via a regular comment.